Dockerfile

```diff
@@ -24,8 +24,9 @@ RUN sed 's/main$/main universe/' -i /etc/apt/sources.list && \
         && \
     rm -rf /var/lib/apt/lists
 
-# Needed for python to consider stdin/stdout as utf8
-RUN locale-gen en_US.UTF-8
+# en_US.UTF-8 is needed for python to consider stdin/stdout as utf8
+# other are custom to provide backward support with scrapy cloud 1.0
+RUN locale-gen en_US.UTF-8 de_DE.UTF-8 nl_NL.UTF-8
```

Comment:
there was a request to add `fr_FR.UTF-8` locale btw https://scrapinghub.slack.com/archives/scrapycloud/p1462896344001198 It was for hworker and we suggested user to use `dateparser`, but having it will be convenient if we are going to have locales for `de` and `nl`